index.php

```diff
@@ -56,16 +106,76 @@ function gutenberg_register_scripts() {
  * @since 0.1.0
  */
 function gutenberg_scripts_and_styles( $hook ) {
-	if ( 'toplevel_page_gutenberg' === $hook ) {
-		// Scripts
-		wp_register_script( 'gutenberg-content', plugins_url( 'post-content.js', __FILE__ ) );
-		wp_enqueue_script( 'wp-editor', plugins_url( 'editor/build/index.js', __FILE__ ), array( 'wp-blocks', 'wp-element', 'gutenberg-content' ), false, true );
-		wp_add_inline_script( 'wp-editor', 'wp.editor.createEditorInstance( \'editor\', { content: window.content } );' );
-
-		// Styles
-		wp_enqueue_style( 'wp-editor-font', 'https://fonts.googleapis.com/css?family=Noto+Serif:400,400i,700,700i', false );
```

Comment:
The third `false` parameter here was incorrect.  This is the [`$deps` parameter](https://developer.wordpress.org/reference/functions/wp_enqueue_style/), which is [set to `array()` if it's not an array](https://github.com/WordPress/wordpress-develop/blob/4.7.3/src/wp-includes/class-wp-dependency.php#L83).